src/components/LogTab/index.jsx

```diff
@@ -81,7 +84,7 @@ export class LogTab extends React.Component {
         this.tail.unwatch()
       }
       catch(error) {
-        console.log(error)
+        logger.error(error.stack)

```

Comment:
Aaah nice.  I was wondering how winston would do something like `logger.exception()` in python.